lib/remote_storage/riak.rb

```diff
@@ -68,7 +68,8 @@ def get_data(user, directory, key)
       when "application/json"
         return object.data.to_json
       else
-        return serializer_for(object.content_type) ? object.data : object.raw_data
+        # Never return nil, always turn data into a string
```

Comment:
Yeah that fix doesn't make sense, thanks. I completely forgot about this pull request. I'm going to do a nil check and return an empty string only when needed.